src/jsxFactory.ts

```diff
@@ -0,0 +1,36 @@
+import {createElement, ReactElement, ReactType} from 'react';
```

Comment:
This file is a copy and paste of https://github.com/cyclejs/react-dom/blob/229e592ab80998ace5bf4fd0c12eee36f4499731/src/jsx-factory.ts, except for import path of `incorporate` funciton.
The original filename was kebab case, but I've renamed to camel case, following the naming convention of this repository.